app/service/rest_svc.py

```diff
@@ -304,11 +306,13 @@ async def validate(op):
     async def get_agent_configuration(self, data):
         abilities = await self.get_service('data_svc').locate('abilities', data)
 
-        raw_abilities = [{'platform': ability.platform, 'executor': ability.executor,
-                          'description': ability.description, 'command': ability.raw_command,
-                          'variations': [{'description': v.description, 'command': v.raw_command}
-                                         for v in ability.variations]}
-                         for ability in abilities]
+        raw_abilities = []
+        for ability in abilities:
+            for executor in ability.executors:
+                variations = [{'description': v.description, 'command': v.raw_command} for v in executor.variations]
```

Comment:
And so shortly after I write a comment about them, I wild dictionary literal appears!